browser/ui/webui/new_tab_page/brave_new_tab_page_handler.cc

```diff
@@ -73,30 +73,55 @@ void BraveNewTabPageHandler::ChooseLocalCustomBackground() {
 
 void BraveNewTabPageHandler::UseBraveBackground() {
   // Call ntp custom background images service.
-  profile_->GetPrefs()->SetBoolean(kNewTabPageCustomBackgroundEnabled, false);
+  auto* prefs = profile_->GetPrefs();
+  prefs->SetBoolean(kNewTabPageCustomBackgroundEnabled, false);
+  prefs->SetBoolean(kNewTabPageSolidColorBackgroundEnabled, false);
   OnCustomBackgroundImageUpdated();
   DeleteSanitizedImageFile();
 }
 
-bool BraveNewTabPageHandler::IsCustomBackgroundEnabled() const {
+void BraveNewTabPageHandler::UseSolidColorBackground(const std::string& color) {
+  profile_->GetPrefs()->SetBoolean(kNewTabPageCustomBackgroundEnabled, false);
+  profile_->GetPrefs()->SetBoolean(kNewTabPageSolidColorBackgroundEnabled,
+                                   true);
+  profile_->GetPrefs()->SetString(kNewTabPageSelectedSolidColorBackground,
+                                  color);
+  OnCustomBackgroundImageUpdated();
```

Comment:
Let's call `DeleteSanitizedImageFile()`. If previously used setting was custom image, that file should be deleted.

Reply:
Good. Done.